framework/packages/abstract-interface/src/deployment.rs

```diff
@@ -354,7 +354,7 @@ mod test {
     fn have_some_state() {
         State::get("state.json").unwrap();
         let state = State::load_state();
-        let vc_juno = &state["juno-1"]["code_ids"].get(REGISTRY);
+        let vc_juno = &state["pion-1"]["code_ids"].get(ANS_HOST);
```

Comment:
That's doesn't seem to be the contract you're trying to query (or mayeb the var name is not good)